front/src/Phaser/Game/GameScene.ts

```diff
@@ -95,10 +103,16 @@ export class GameScene extends Phaser.Scene implements GameSceneInterface{
         //add layer on map
         this.Layers = new Array<Phaser.Tilemaps.StaticTilemapLayer>();
         let depth = -2;
-        this.map.layers.forEach((layer) => {
+        this.map.layers.forEach((layer : ITiledMapLayer) => {
             if (layer.type === 'tilelayer') {
                 this.addLayer(this.Map.createStaticLayer(layer.name, this.Terrains, 0, 0).setDepth(depth));
             }
+            if (layer.type === 'tilelayer' && layer.name === "exit") {
+                this.loadNextGame(layer);
```

Comment:
:+1: yay!

Reply:
We should document that in the README (how to create a map with a start and an exit) => this is needed if we want some help for the maps